.github/workflows/check-dev-tools.yml

```diff
@@ -21,10 +21,10 @@ jobs:
     - uses: actions/setup-go@v5
       with:
         go-version-file: .go-version
-    - name: Fix Code is not compatible with Python 3.12
-      uses: actions/setup-python@v5
+    - uses: actions/setup-python@v5
       with:
-        python-version: '3.10'
+        cache: "pip"
+        python-version-file: .python-version
```

Comment:
Check seems to have run without issues with the bundled python-version -> https://github.com/elastic/beats/actions/runs/13768950759/job/38502443354?pr=43149